src/utils/PaymentsConverter.ts

```diff
@@ -0,0 +1,168 @@
+    causaleVersamento: datiPagamentoPA.causaleVersamento,
+    enteBeneficiario: {
+      codiceIdentificativoUnivoco:
+        datiPagamentoPA.enteBeneficiario.identificativoUnivocoBeneficiario,
+      denominazioneBeneficiario:
+        datiPagamentoPA.enteBeneficiario.denominazioneBeneficiario,
+      codiceUnitOperBeneficiario:
+        datiPagamentoPA.enteBeneficiario.codiceUnitOperBeneficiario,
+      denomUnitOperBeneficiario:
+        datiPagamentoPA.enteBeneficiario.denomUnitOperBeneficiario,
+      indirizzoBeneficiario:
+        datiPagamentoPA.enteBeneficiario.indirizzoBeneficiario,
+      civicoBeneficiario: datiPagamentoPA.enteBeneficiario.civicoBeneficiario,
+      capBeneficiario: datiPagamentoPA.enteBeneficiario.capBeneficiario,
+      localitaBeneficiario:
+        datiPagamentoPA.enteBeneficiario.localitaBeneficiario,
+      provinciaBeneficiario:
+        datiPagamentoPA.enteBeneficiario.provinciaBeneficiario,
+      nazioneBeneficiario: datiPagamentoPA.enteBeneficiario.nazioneBeneficiario
+    },
+    spezzoniCausaleVersamento: datiPagamentoPA.spezzoniCausaleVersamento
+  }).mapLeft(() => {
+    return new Error();
+  });
+}
+
+// Convert PaymentsActivationRequest (controller) to PaymentsActivationRequestPagoPa (PagoPa API)
+export function getPaymentsActivationRequestPagoPa(
+  pagoPaConfig: PagoPaConfig,
+  paymentsActivationRequest: PaymentsActivationRequest
+): Either<Error, InodoAttivaRPTInput> {
+  // TODO: [#158209998] Remove try\catch and replace it with decode when io-ts types will be ready
+  try {
+    return right({
+      identificativoPSP: pagoPaConfig.IDENTIFIER.IDENTIFICATIVO_PSP,
+      identificativoIntermediarioPSP:
+        pagoPaConfig.IDENTIFIER.IDENTIFICATIVO_INTERMEDIARIO_PSP,
+      identificativoCanale: pagoPaConfig.IDENTIFIER.IDENTIFICATIVO_CANALE,
+      password: pagoPaConfig.IDENTIFIER.TOKEN,
+      codiceContestoPagamento:
+        paymentsActivationRequest.codiceContestoPagamento,
+      identificativoIntermediarioPSPPagamento:
+        pagoPaConfig.IDENTIFIER.IDENTIFICATIVO_INTERMEDIARIO_PSP,
+      identificativoCanalePagamento:
+        pagoPaConfig.IDENTIFIER.IDENTIFICATIVO_CANALE,
+      codificaInfrastrutturaPSP: codificaInfrastrutturaPSPEnum.QR_CODE,
+      codiceIdRPT: {
+        CF: paymentsActivationRequest.codiceIdRPT.CF,
+        AuxDigit: paymentsActivationRequest.codiceIdRPT.AuxDigit,
+        CodStazPA: paymentsActivationRequest.codiceIdRPT.CodStazPA,
+        CodIUV: paymentsActivationRequest.codiceIdRPT.CodIUV
+      },
+      datiPagamentoPSP: {
+        importoSingoloVersamento:
+          paymentsActivationRequest.importoSingoloVersamento
+      }
+    });
+  } catch (exception) {
+    return left(new Error());
+  }
+}
+
+// Convert PaymentsActivationResponsePagoPa (PagoPa API) to PaymentsActivationkResponse (controller)
+export function getPaymentsActivationResponse(
+  iNodoAttivaRPTOutput: InodoAttivaRPTOutput
+  // tslint:disable-next-line:readonly-array
```

Comment:
remove the tslint ignore and use `Validation<PaymentsActivationResponse>` as return type, see [here](https://github.com/gcanti/io-ts/blob/master/src/index.ts#L22)